api/kyverno/v2beta1/cleanup_policy_types.go

```diff
@@ -224,9 +224,13 @@ type CleanupPolicySpec struct {
 	// Conditions defines the conditions used to select the resources which will be cleaned up.
 	// +optional
 	Conditions *AnyAllConditions `json:"conditions,omitempty"`
+
+	// DeletionPropagationPolicy defines how resources will be deleted (Foreground, Background).
+	// +optional
+	DeletionPropagationPolicy *string `json:"deletionPropagationPolicy,omitempty"`
```

Comment:
why is it a pointer to a string?

Reply:
@eddycharly commented that this should be a pointer

Reply:
Yeah, a pointer but not to a string ;-)

Reply:
okay

Reply:
Use the `metav1` underlying type directly.

Reply:
Okay thanks